README.md

```diff
@@ -1,13 +1,66 @@
 # Code Samples for DevNet Express Cloud Collaboration tracks
 
-This repository holds code samples for these two DevNet Express tracks:
+These self-paced interactive tutorials provide instructions for developers to read and use code examples.
```

Comment:
Let's leave this README alone except for light edits, because this repo does not contain Learning Labs content.